docs/Users_Guide/overview.rst

```diff
@@ -95,23 +95,24 @@ METplus Wrappers is an evolving application. New capabilities are
 planned in controlled, successive version releases that are synchronized
 with MET releases. Bug fixes and user-identified problems will be
```

Comment:
We don't really have a "known issues" section on the website anymore. Recommend the following wording instead:
```
Software bugs and user-identified problems will be documented using GitHub issues and will be fixed either in the next bugfix or official release.
```

Reply:
@JohnHalleyGotway Thanks so much for catching that!  My vision was narrow.  I didn't read the surrounding text and only focused in on met_help.  I'll expand my vision for changes in the other repos.